### DISCUSSION TITLE
Root access

### DISCUSSION THREAD

**Discussion by TribunalGS** [bmax121/APatch#307] [Q&A]
Hello. Is it possible in principle to create a redirection of root rights requests from Magisk to Apatch? Or, for example, a module that will intercept such requests and convert them.

**Comment by bmax121** [comment]
Installed both magisk and apatch?

**Reply by TribunalGS** [reply]
Sorry, APatch is installed. By this question I meant is it possible to change the path to binaries through an overlay? For example, if the application requires Magisk, but ignores the rights issued by the Apatch. This was explained to me by the fact that the location of the binaries is different.
